app/src/main/java/org/eu/exodus_privacy/exodusprivacy/manager/storage/ExodusDataStoreModule.kt

```diff
@@ -0,0 +1,37 @@
+package org.eu.exodus_privacy.exodusprivacy.manager.storage
+
+import androidx.datastore.preferences.core.Preferences

```

Comment:
@PatDyn can you check if import is correct because linter doesn't like wildcard import